modules/sbo-release-notes-1-2.adoc

```diff
@@ -0,0 +1,62 @@
+[id="sbo-release-notes-1-2_{context}"]
+// Module included in the following assembly:
+//
+// * applications/connecting_applications_to_services/sbo-release-notes.adoc
+:_content-type: REFERENCE
+= Release notes for {servicebinding-title} 1.2
+
+{servicebinding-title} 1.2 is now available on {product-title} 4.7, 4.8, 4.9, 4.10, and 4.11.
+
+[id="new-features-1-2_{context}"]
+== New features
+This section highlights what is new in {servicebinding-title} 1.2:
+
+* Enable {servicebinding-title} to consider optional fields in the annotations by setting the `optional` flag value to `true`.
+* Support for `servicebinding.io/v1beta1` resources.
+* Improvements to the discoverability of bindable services by exposing the relevant binding secret without requiring a workload to be present.
+
+[id="known-issues-1-2_{context}"]
+== Known issues
+* By default, the projected files get their permissions set to 0644. {servicebinding-title} cannot set specific permissions due to a bug in Kubernetes that causes issues if the service expects specific permissions such as, `0600`. As a workaround, you can modify the code of the program or the application that is running inside a workload resource to copy the file to the `/tmp` directory and set the appropriate permissions. link:https://issues.redhat.com/browse/APPSVC-1127[APPSVC-1127]
```

Comment:
💡 RedHat.Usage 15(124-130) Verify your use of 'due to' with the word usage guidelines.
💡 RedHat.SimpleWords 15(256-262) Use simple language. Consider using 'change' rather than 'modify'.